packages/dcos-log/buildinfo.json

```diff
@@ -0,0 +1,11 @@
+{
+  "docker": "dcos/dcos-builder:dcos-log_dockerdir-latest",
```

Comment:
we are using this approach to build [`dcos-ca`](https://github.com/mesosphere/dcos-enterprise/blob/master/packages/dcos-ca/buildinfo.json)